lpa-submissions-e2e-tests/cypress/support/apis/appeal-reply-service-api/index.js

```diff
@@ -0,0 +1,59 @@
+const create = (appealId) =>{
```

Comment:
a bunch of helpful cypress commands that do the posting+putting

i left it in the shape that worked for me; there may be better shapes

i eventually realised that the best bet is to do the ".as('appeal')" stuff here and just pull it out in the step-def.. means we can set up several aliases under the covers here without exposing the whole post/put/post/put dance to the folk writing tests...